templates/template-next-tailwind-basic/anchor/tests/basic.test.ts

```diff
@@ -1,7 +1,16 @@
+import * as anchor from '@coral-xyz/anchor'
+import { Program } from '@coral-xyz/anchor'
```

Comment:
The issue #58 mentions to implement them `using the code generated by codama.`, so we don't want it to use `@coral-xyz/anchor` as we did in the legacy templates.

The code generated by `codama` works with the modern Solana SDKs (`@solana/kit` and `gill`).